lib/tasks/pact.rake

```diff
@@ -1,5 +1,17 @@
 return if Rails.env.production?
 
 require "pact/tasks"
-require "pact_broker/client/tasks"
 require "pact/tasks/task_helper"
+
+desc "Verify that Frontend honours all contracts in the pactfile created by a given branch of gds-api-adapters"
+task "pact:verify:branch", [:branch_name] => :environment do |t, args|
```

Comment:
This task also isn't necessary. Removed in https://github.com/alphagov/frontend/pull/2712.